src/sampling.jl

```diff
@@ -74,12 +74,23 @@ end
 
 ### draw a pair of distinct integers in [1:n]
 
+"""
+    samplepair(n) -> (Int, Int)
+
+Draw a pair of distinct integers between 1 and `n`.
```

Comment:
Maybe put "distinct" in italics (`*`)? I always miss that detail about that function.